sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Changed return type of `DataLakeFileSystemClient::GetProperties` to `DataLakeFileSystemProperties`.
 - Changed return type of `DataLakePathClient::GetProperties` to `DataLakePathProperties`.
 - Removed `GetUserDelegationKeyResult`.
+- Removed `RequestId` from the returned results.
```

Comment:
nit: wording consistency across blobs and datalake (see https://github.com/Azure/azure-sdk-for-cpp/pull/1931)
```suggestion
- Removed `RequestId` from return types.
```